.gitignore

```diff
@@ -2,4 +2,6 @@
 Cargo.lock
 *.sqlite
 *.sqlite-*
-.DS_Store
\ No newline at end of file
+.DS_Store
+.env
```

Comment:
```suggestion
```

I removed this from .gitignore in my last PR. Delete this line or rebase onto main?